lib/biker.rb

```diff
@@ -1,3 +1,15 @@
 class Biker
+    attr_reader :name, :acceptable_terrain, :max_distance
 
-end
\ No newline at end of file
+    def name
+    end 
+    
+    def acceptable_terrain
+    end
+
+    def max_distance
+    end
+end
+
+biker1 = Biker1.new({name: "Kenny", acceptable_terrain: ["gravel", "hills"] , max_distance: 30})
```

Comment:
We don't need to instantiate an object here. We just want the class definition above